src/panoptes/pocs/filterwheel/filterwheel.py

```diff
@@ -196,28 +213,53 @@ def move_to(self, position, blocking=False):
             self.logger.error(msg)
             raise error.PanError(msg)
 
-        position = self._parse_position(position)
-        self.logger.info("Moving {} to position {} ({})".format(
-            self, position, self.filter_names[position - 1]))
+        # Will raise a ValueError at this point if new_position is not a valid position
+        new_position = self._parse_position(new_position)
 
-        if position == self.position:
+        if new_position == self.position:
             # Already at requested position, don't go nowhere.
+            self.logger.debug(f"{self} already at position {new_position}" + \
+                              f" ({self.filter_name(new_position)})")
             return self._move_event
 
+        if new_position == self._dark_position:
+            # Moving from light into darkness... Store current position so we can revert
+            # back to it if requested with move_to_light_position()
+            self._last_light_position = self.position
```

Comment:
I just wondering if we should have a `previous_filter` or even `filter_history`. :thinking: A different PR though.

Reply:
I think the nomenclature is OK, for now.